packages/remark-parse/lib/tokenizer.js

```diff
@@ -304,7 +304,7 @@ function mergeText(prev, node) {
 
 // Merge two blockquotes: `node` into `prev`, unless in CommonMark mode.
```

Comment:
It looks like this comment needs to be updated

```suggestion
 // Merge two blockquotes: `node` into `prev`, unless in CommonMark or gfm modes.
```

Reply:
Absolutely !